impeller/entity/contents/filters/gaussian_blur_filter_contents.cc

```diff
@@ -216,11 +222,11 @@ std::optional<Snapshot> DirectionalGaussianBlurFilterContents::RenderFilter(
   };
 
   Scalar x_scale =
-      std::min(1.0, 1.0 / std::ceil(std::log2(transformed_blur_radius_length)));
-  auto out_texture =
-      renderer.MakeSubpass(ISize(pass_texture_rect.size.width * x_scale,
-                                 pass_texture_rect.size.height),
-                           callback);
+      1.0 /
+      std::ceil(std::log2(std::max(2.0f, transformed_blur_radius_length)));
```

Comment:
Looking at this line, I'm missing the difference from the old code. Is it impacting the solution to the jitter?

Reply:
The old code could result in -inf. This code won't.

Reply:
D'oh! Yep, I see it...